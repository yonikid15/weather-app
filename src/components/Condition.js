import React from 'react';

const Condition = ({ condition }) => {
  return (
    <div>
      <h3>
          <span>Condition: </span>
          <span>{condition}</span>
      </h3>
    </div>
  );
};

export default Condition;