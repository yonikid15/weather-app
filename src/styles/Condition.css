.condition {
  grid-column: 6 / -1;
  grid-row: span 2;
  background-color: rgba(0, 0, 0, 0.233);
  border-radius: 10px;
  text-align: center;
  box-shadow: 0 19px 38px rgba(0, 0, 0, 0.3), 0 15px 12px rgba(0, 0, 0, 0.22);

  &__title {
    font-weight: 500;
    font-size: 25px;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-content: center;
  }

  &__icon {
    padding: 10px;
    font-size: 85px;
  }
}