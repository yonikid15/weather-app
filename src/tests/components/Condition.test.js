import React from 'react';
import { shallow } from 'enzyme';
import Condition from '../../components/Condition';

test('should render Condition correctly', () => {
  const wrapper = shallow( <Condition condition={'cloudy'} /> );
  expect( wrapper ).toMatchSnapshot();
});