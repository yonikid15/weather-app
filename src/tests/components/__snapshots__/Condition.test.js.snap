// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Condition correctly 1`] = `
<div
  className="condition"
>
  <h3
    className="condition__title"
  >
    <i
      className="wi wi-cloudy condition__icon"
    />
    <span>
      cloudy
    </span>
  </h3>
</div>
`;
